config/v1/types_cluster_version.go

```diff
@@ -806,6 +842,15 @@ type ConditionalUpdate struct {
 // for not recommending a conditional update.
 // +k8s:deepcopy-gen=true
 type ConditionalUpdateRisk struct {
+	// conditions represents the observations of the conditional update
+	// risk's current status. Known types are:
+	// * Apply, for whether the risk is applied to the current cluster.
```

Comment:
`Applied` feels a bit more appropriate here than `Apply`

Reply:
going back through the resolved threads to see what I'd missed, I see this one asking for `Applied`.  [I later asked for (and Hongkai pivoted to) `Applies`][1].  Hopefully that's also ok with you, @everettraven 😅 

[1]: https://github.com/openshift/api/pull/2360#discussion_r2162586035